go.mod

```diff
@@ -14,9 +14,10 @@ require (
 	github.com/sirupsen/logrus v1.8.1
 	github.com/spf13/cobra v1.1.3
 	github.com/spf13/pflag v1.0.5
-	github.com/tidwall/gjson v1.7.4 // indirect
+	github.com/tidwall/gjson v1.8.1 // indirect
 	golang.org/x/crypto v0.0.0-20200622213623-75b288015ac9
-	golang.org/x/term v0.0.0-20210317153231-de623e64d2a6
-	gopkg.in/check.v1 v1.0.0-20190902080502-41f04d3bba15 // indirect
+	golang.org/x/term v0.0.0-20201126162022-7de9c90e9dd1
 	gopkg.in/yaml.v2 v2.4.0
 )
+
+replace github.com/mesosphere/konvoy-image-builder/printerlib => ./printerlib
```

Comment:
Why is this needed?

Reply:
removed, it was for constant error, but really no need to use it